Jetcaster/README.md

```diff
@@ -3,60 +3,34 @@
 # Jetcaster sample 🎙️
 
 Jetcaster is a sample podcast app, built with [Jetpack Compose][compose]. The goal of the sample is to
-showcase dynamic theming and full featured architecture.
+showcase building with Compose across multiple form factors (mobile, TV, and Wear) and full featured architecture.
 
 To try out this sample app, use the latest stable version
 of [Android Studio](https://developer.android.com/studio).
 You can clone this repository or import the
 project from Android Studio following the steps
 [here](https://developer.android.com/jetpack/compose/setup#sample).
 
-### Status: 🚧 In progress 🚧
-
-Jetcaster is still in the early stages of development, and as such only one screen has been created so far. However,
-most of the app's architecture has been implemented, as well as the data layer, and early stages of dynamic theming.
-
-
 ## Screenshots
 
-<img src="docs/screenshots.png"/>
+![readme_cover](https://github.com/android/compose-samples/assets/10263978/a58ab950-71aa-48e0-8bc7-85443a1b4f6b)
 
 ## Features
 
-This sample contains 2 screens so far: the home screen, and a player screen.
+This sample has 3 components: the home screen, the podcast details screen, and the player screen
```

Comment:
We could potentially have different features for form factors, e.g. Wear does not deal with multiple panels. We can do in another PR.

Reply:
Yes, we can describe the differences of each form factor in more detail. I agree that we should tackle that in a separate PR.